synapse/models/geopol.py

```diff
@@ -42,7 +48,77 @@ def getModelDefs(self):
+                    )),
+                    ('pol:office', {}, (
+                        ('title', ('ou:jobtitle', {}), {
+                            'ex': 'united states senator',
+                            'doc': 'The title of the political office.'}),
+                        ('position', ('ou:position', {}), {
+                            'doc': 'The position this office holds in the org chart for the governing body.'}),
+                        ('termlimit', ('int', {}), {
+                            'doc': 'The maximum number of times a single person may hold the office.'}),
+                        ('govbody', ('ou:org', {}), {
+                            'doc': 'The governmental body which contains the office.'}),
+                    )),
+                    ('pol:term', {}, (
+                        ('office', ('pol:office', {}), {
+                            'doc': 'The office held for the term.'}),
+                        ('start', ('time', {}), {
+                            'doc': 'The start of the term of office.'}),
+                        ('end', ('time', {}), {
+                            'doc': 'The of the term of office.'}),
+                        ('election', ('pol:election', {}), {
+                            'doc': 'The election that determined who held office during the term.'}),
+                        ('contact', ('ps:contact', {}), {
+                            'doc': 'The contact information of the person who held office during the term.'}),
+                        ('party', ('ou:org', {}), {
+                            'doc': 'The political party of the person who held office during the term.'}),
+                    )),
+                    ('pol:candidate', {}, (
+                        ('contact', ('ps:contact', {}), {
+                            'doc': 'The contact information of the candidate.'}),
+                        ('race', ('pol:race', {}), {
+                            'doc': 'The race the candidate is participating in.'}),
+                        ('campaign', ('ou:campaign', {}), {
+                            'doc': 'The official campaign to elect the candidate.'}),
+                        ('winner', ('bool', {}), {
+                            'doc': 'Records the outcome of the election.'}),
+                        ('party', ('ou:org', {}), {
+                            'doc': 'The declated political party of the candidate.'}),
+                    )),
+                    ('pol:pollingplace', {}, (
+                        ('election', ('pol:election', {}), {
+                            'doc': 'The election that the polling place is designated for.'}),
+                        ('name', ('geo:name', {}), {
+                            'doc': 'The name of the polling place at the time. This may differ from the official place name.'}),
+                        ('place', ('geo:place', {}), {
+                            'doc': 'The place where votes were cast.'}),
```

Comment:
"Place" - should this be "The location of the polling place" vs "place where votes were cast"? Bit of mental dissonance when reading this definition. E.g., I can "cast my vote" by dropping my ballot at a designated dropoff location, which is not a polling place (i.e., with voting machines, etc.)